mlrun/utils/http.py

```diff
@@ -114,6 +114,12 @@ def request(self, method, url, **kwargs):
         )
         while True:
             try:
+                # import here to prevent import cycle
+                from mlrun.config import config as mlconf
+
+                verify_ssl = mlconf.httpdb.http.verify
+                if not verify_ssl:
+                    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
This is duplicated from `datastore/base.py`. move it to a common location and call it from both places